agent/engine/docker_container_engine.go

```diff
@@ -48,8 +47,8 @@ const (
 const (
 	pullImageTimeout        = 2 * time.Hour
 	createContainerTimeout  = 3 * time.Minute
-	startContainerTimeout   = 1*time.Minute + 30*time.Second
-	stopContainerTimeout    = 1 * time.Minute
+	startContainerTimeout   = 1 * time.Minute + 30*time.Second
+	stopContainerTimeout    = 30 * time.Second
```

Comment:
It might be more clear to change this name to `stopContainerAPITimeout`.  Also, can you run `gofmt` on this file?